interface/forms/newpatient/common.php

```diff
@@ -86,16 +86,35 @@ function refreshIssue(issue, title) {
   s.options[s.options.length] = new Option(title, issue, true, true);
  }
 
+ <?php
```

Comment:
We agree that having a standard form submission in all openemr is the best practice, but currently we are "short-handed" and we can't do this refactor now as it will require a lot of QA and dev time. Maybe we can release this feature as is for now? And as soon as we/community have some extra time we can change it. 
Maybe we miss understood what you want, if you think so please explain better.